samples/cpp/qrcode_multi.cpp

```diff
@@ -0,0 +1,196 @@
+#include "opencv2/objdetect.hpp"
```

Comment:
We don't need new sample.

Please update existing one:
- support both modes
- switch between modes by "space" key